README.md

```diff
@@ -1,3 +1,26 @@
 ## Experimenter Docs
 
 This repo is the documentation hub for [Experimenter](https://github.com/mozilla/experimenter).
+
+## Working on the docs
+
+Pages is written in Markdown, and can be found under [`/docs`](/docs).
+
+### Using GitHub
+
+You do not necessarily need to clone or set up this repo locally if you just want to contribute to or change documentation.
+
+If you wish to modify an existing page, navigate to its Markdown file and click the "Edit" pencil icon in the top-right of the content. You will be presented with a Markdown editor. Make and commit your changes.
```

Comment:
We probably want mention & suggestion to submit PRs from edits? Don't think we'll want folks working directly on main branch.

Been awhile since I did PRs purely from the github web UI, but I think you can edit markdown in-browser to create PRs

Reply:
Yeah this write-up definitely needs some finessing - this is kind of just an initial go at it. I want to go through the flow (of creating new and editing existing MD files) myself and then I can more accurately update this.

Reply:
FWIW, we'll be writing "official" docs on this in EXP-1027